README.md

```diff
@@ -1,32 +1,47 @@
-# smooth-operator
-Shared logic and structure library for PDOK operators. 
-This repo contains global Kinds and shared code base for other operator.
-
 <p align="center">
     <a href="https://youtu.be/4TYv2PhG89A?si=X_a7zpjaD_esgUAO&t=74">
         <img src="docs/mascotte.png" alt="Smooth-operator" title="Smooth-operator" width="300" />
     </a>
 </p>
 
+# Smooth Operator
+This repository contains code shared by the Kubernetes operators developed by [PDOK](https://www.pdok.nl/). There are no executable programs present in this codebase. 
+The goal of this repository is to support the development work of PDOK, but it can also be used as a starter to develop custom map related operators. At the moment the following operators have this repository as a basis: 
+- [Atom operator](https://github.com/PDOK/atom-operator), an operator for deploying atomfeed instances
+- [Mapserver operator](https://github.com/PDOK/mapserver-operator), an operator for deploying mapserver instances
+
+Although this repository is oriented towards geoinformation related operators, it can be used as a base for non-geoinformation operators.
```

Comment:
Zou deze zin weghalen. Zelf gaan we dit ook alleen in geo-related operators gebruiken, dus zie niet echt een use case hiervoor.